packages/loaders/examples/skeleton.md

```diff
@@ -18,13 +18,13 @@ const StyledCol = styled(Col)`
   <Row alignItems="start">
     <StyledCol md={8} isDarkMode={state.isDarkMode}>
       <XXXL>
-        {state.isLoading ? <Skeleton isDark={state.isDarkMode} /> : 'There. You see Lord Vader.'}
+        {state.isLoading ? <Skeleton isLight={state.isDarkMode} /> : 'There. You see Lord Vader.'}
       </XXXL>
       {state.isLoading && (
         <MD>
-          <Skeleton isDark={state.isDarkMode} />
-          <Skeleton isDark={state.isDarkMode} />
-          <Skeleton isDark={state.isDarkMode} />
+          <Skeleton isLight={state.isDarkMode} />
```

Comment:
With the new naming we now have `light == dark`. Should we rename this examples state to align with the new prop name?

Reply:
I decided not to. For now, the example shows using `isLight` in a dark mode setting.